core/src/main/scala/zio/logging/Logging.scala

```diff
@@ -0,0 +1,39 @@
+package zio.logging
+
+import zio.{ Cause, FiberRef, UIO, ZIO }
+
+trait AbstractLogging[Message] {
+  def logging: AbstractLogging.Service[Any, Message]
+}
+
+object AbstractLogging {
+
+  trait Service[-R, Message] {
+    def context: LoggingContext[Message]
```

Comment:
I might call this `loggingContext` just to disambiguate.

Reply:
good point